key_rings.go

```diff
@@ -0,0 +1,75 @@
+package kp
+
+import (
+	"context"
+	"fmt"
+	"time"
+)
+
+const (
+	path = "key_rings"
+)
+
+type Ring struct {
```

Comment:
```suggestion
type KeyRing struct {
```

I think its better to use the full name